components/clp-package-utils/clp_package_utils/scripts/compress_from_s3.py

```
@@ -222,8 +224,12 @@ def main(argv):
 
     try:
         config_file_path = pathlib.Path(parsed_args.config)
-        clp_config = load_config_file(config_file_path, default_config_file_path, clp_home)
-        clp_config.validate_logs_dir()
+        clp_config = load_config_file(
+            resolve_host_path(config_file_path),
+            resolve_host_path(default_config_file_path),
+            clp_home,
+        )
+        clp_config.validate_logs_dir(True)
```

Ruff (0.14.2)

Boolean positional value in function call

(FBT003)